processor/simple_supply_tp/handler.py

```diff
@@ -72,6 +77,36 @@ def _create_agent(state, public_key, payload):
         timestamp=payload.timestamp)
 
 
+def _create_record(state, public_key, payload):
+    if state.get_agent(public_key) is None:
+        raise InvalidTransaction('Agent with the public key {} does '
+                                 'not exist'.format(public_key))
+
+    if payload.data.record_id == '':
+        raise InvalidTransaction('No record ID provided')
+
+    if state.get_record(payload.data.record_id):
+        raise InvalidTransaction('Identifier {} belongs to an existing '
+                                 'record'.format(payload.data.record_id))
+
+    _validate_latlng(payload.data.latitude, payload.data.longitude)
+
+    state.set_record(
+        public_key=public_key,
+        latitude=payload.data.latitude,
+        longitude=payload.data.longitude,
+        record_id=payload.data.record_id,
+        timestamp=payload.timestamp)
+
+
+def _validate_latlng(latitude, longitude):
+    LOGGER.info('latitude: %s', latitude)
```

Comment:
This seems like it should be at most debug level, not info.

Reply:
Accidentally forgot to delete it when I was debugging. I will fix that